packages/code-studio/vite.config.ts

```diff
@@ -19,13 +19,6 @@ export default defineConfig(({ mode }) => {
   // These are paths which should be proxied to the core server
   // https://vitejs.dev/config/server-options.html#server-proxy
   const proxy = {
-    // Proxy styleguide here instead of as a route in our app router
-    // That way, it is not included in the production build
```

Comment:
We need to update the `README.md` in `packages/code-studio` to fix the link to the styleguide; right now it just has http://localhost:4000/styleguide, should be http://localhost:4000/ide/styleguide

Also noting that this does not affect bundle size since you're using code splitting to only load the style guide when we go to that route (I think...), _however_ this also means that anybody can open up the styleguide. Which I guess is a good thing if we want people to develop their own themes.

Reply:
FWIW, @dsmmcken is not a fan of surfacing the styleguide to users. I don't think it would be hard to conditionally show it based on an env var + vite config